packages/launcher/src/renderer/app/AppContainer.js

```diff
@@ -7,6 +7,8 @@ import type { ID } from '@mainframe/utils-id'
 import React, { Component } from 'react'
 import { View, StyleSheet } from 'react-native-web'
 
+import styled from 'styled-components/native'
+
 import colors from '../colors'
 import Text from '../UIComponents/Text'
```

Comment:
We should probably update these to use Morpheus UI and maybe change the views with styles also to styled components so we don't have a mix of the two types.